client/components/site-verticals-suggestion-search/index.jsx

```diff
@@ -102,9 +102,7 @@ export class SiteVerticalsSuggestionSearch extends Component {
 		return (
 			<SuggestionSearch
 				id="siteTopic"
-				placeholder={
-					placeholder || translate( 'e.g. Fashion, travel, design, plumber, electrician' )
-				}
+				placeholder={ placeholder || translate( 'e.g. Fashion, travel, design, plumber' ) }
```

Comment:
:disappointed: 23 existing translations will be lost with this change.

Reply:
If we're going to change this translation, should we make `e.g` consistent? `E.g.,` or `e.g` - you choose :) because I can't find a definitive example: they're both peppered about the place.